scripts/release/mule/package/deb/package.sh

```diff
@@ -125,11 +126,46 @@ Copyright: Algorand developers <dev@algorand.com>
 License: AGPL-3+
 EOF
 
-sed 's/^$/./g' < COPYING | sed 's/^/ /g' >> "${PKG_ROOT}/DEBIAN/copyright"
-mkdir -p "${PKG_ROOT}/usr/share/doc/${ALGORAND_PACKAGE_NAME}"
-cp -p "${PKG_ROOT}/DEBIAN/copyright" "${PKG_ROOT}/usr/share/doc/${ALGORAND_PACKAGE_NAME}/copyright"
+sed 's/^$/./g' < COPYING | sed 's/^/ /g' >> "$PKG_ROOT/DEBIAN/copyright"
+mkdir -p "$PKG_ROOT/usr/share/doc/$ALGORAND_PACKAGE_NAME"
+cp -p "$PKG_ROOT/DEBIAN/copyright" "$PKG_ROOT/usr/share/doc/$ALGORAND_PACKAGE_NAME/copyright"
 
-dpkg-deb --build "${PKG_ROOT}" "${OUTPUT_DEB}"
+dpkg-deb --build "$PKG_ROOT" "$OUTPUT_DEB"
+
+############################################################
+
+pushd "$PKG_DIR"
+
+STATUSFILE=build_status_${CHANNEL}_${VERSION}
+
+cat << EOF >> "$STATUSFILE"
+
+go version:
+EOF
+
+/usr/local/go/bin/go version >> "$STATUSFILE"
+
+############################################################
+
+cat << EOF >> "$STATUSFILE"
+
+go env:
+EOF
+
+/usr/local/go/bin/go env >> "$STATUSFILE"
+
+############################################################
+
+cat << EOF >> "$STATUSFILE"
+
+dpkg-l:
+EOF
+
+dpkg -l >> "$STATUSFILE"
+
+popd
+
+############################################################
```

Comment:
Here we need to capture the details of the build env for the `build_status` file.